staging/src/k8s.io/pod-security-admission/cmd/webhook/server/server.go

```diff
@@ -58,6 +58,13 @@ Security Standards.`,
 		},
 		Args: cobra.NoArgs,
 	}
+	// This command does *not* normalize the flags that it inherits, in
+	// contrast to other commands.
+	//
+	// The code for it, pflag.CommandLine.SetNormalizeFunc in
+	// cmd/webhook/webhook.go, didn't have an effect and now the flags with
```

Comment:
I don't understand this - what does it mean it didn't have an effect?  The goal of normalizing is to allow `foo-bar` and `foo_bar` as the same thing

Reply:
The code which constructed the command and its flag sets didn't use the normalization function from `pflag.CommandLine`.